test/loading.jl

```diff
@@ -231,7 +231,7 @@ end
         pkg = recurse_package(n...)
         @test pkg == PkgId(UUID(uuid), n[end])
         @test joinpath(@__DIR__, normpath(path)) == locate_package(pkg)
-        @test Base.compilecache_path(pkg) == Base.compilecache_path(pkg)
+        @test Base.compilecache_path(pkg, UInt64(0)) == Base.compilecache_path(pkg, UInt64(0))
```

Comment:
I took the liberty of changing the type here. That said, is this really the correct test? Should the arguments be different?

Reply:
Yeah, the test is to ensure that the function is deterministic. :)

Reply:
Ref https://github.com/JuliaLang/julia/issues/38101